subsys/mgmt/mcumgr/grp/fs_mgmt/src/fs_mgmt.c

```diff
@@ -940,28 +940,33 @@ static const struct mgmt_handler fs_mgmt_handlers[] = {
 	[FS_MGMT_ID_FILE] = {
 		.mh_read = fs_mgmt_file_download,
 		.mh_write = fs_mgmt_file_upload,
+		.use_custom_payload = false,
```

Comment:
These shouldn't be needed as the struct should default to 0 (false) when not set

Reply:
Removed